mackerel-plugin-solr/README.md

```diff
@@ -37,3 +37,32 @@ command = "/path/to/mackerel-plugin-solr -host=192.168.33.10 -port=8984"
 ## Apache Solr official website
 
 - http://lucene.apache.org/solr/
+
+## Test
+
+### Run
+
+```
+$ cd mackerel-plugin-solr/lib/
+$ go test
+```
+
+### Add fixture json
+
+```
+$ docker pull solr:x.x
+$ docker run --name solr_x -d -p 8983:8983 -t solr:x.x
+$ docker exec -it --user=solr solr_x bin/solr create_core -c testcore
+$ cd mackerel-plugin-solr/lib/
+$ curl -s -S 'http://localhost:8983/solr/admin/cores?wt=json'                                        | jq . > stats/x.x.x/cores.json
+$ curl -s -S 'http://localhost:8983/solr/testcore/admin/mbeans?wt=json&stats=true&cat=QUERYHANDLER'  | jq . > stats/x.x.x/query.json
+$ curl -s -S 'http://localhost:8983/solr/testcore/admin/mbeans?wt=json&stats=true&cat=UPDATEHANDLER' | jq . > stats/x.x.x/update.json
```

Comment:
You should add proper `key=` query parameters to the URL and regenerate stats/x.x.x/ANY.json, otherwise unnecessary stats will be parsed from these jsons in tests.
For example, stats/5.5.4/update.json has `stats.solr-mbeans.[0][1][updateHandler][stats][errors]`, so `testcore_errors_updateHandler` metrics will be created.  But in real situation, this field does not exist because `key=` query parameters are provided.